### DISCUSSION TITLE
React-auth-kit  declaration file missing

### DISCUSSION THREAD

**Discussion by Alvi24** [react-auth-kit/react-auth-kit#1482] [Q&A] [answered]
I have installed the lates react-auth-kit version (^3.0.0-alpha.15)  but it isn't working with typescript. It doesn't work even after i installed @types/react-auth-kit

**Comment by darkmatter18** [comment, marked as answer]
Don't use the alpha release. Please wait for a few more days for the stable release of 3.0.0
